source/installation.html.haml

```diff
@@ -19,24 +19,37 @@ title: Installation
             Windows
           %p
             All binary packages are unofficial third-party builds.
+            Except for those linked from the mpv-player/mpv repository,
+            each commit and pull request has associated test builds.
```

Comment:
I don't understand this sentence.

Reply:
I'm trying to say that all binary build are third-party, except those that are in our (mpv-player/mpv) CI artifacts

Reply:
Changed it a little, maybe it's better.